modules/lx-autoscale/watchmaker-lx-autoscale.cfn.json

```diff
@@ -704,6 +732,173 @@
                             }
                         }
                     },
+                    "install-cloudwatch-agent": {
+                        "commands": {
+                            "01-get-cloudwatch-agent": {
+                                "command": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "mkdir -p /etc/cfn/scripts/ &&",
+                                            " aws s3 cp ",
+                                            {
+                                                "Ref": "CloudWatchAgentUrl"
+                                            },
+                                            " /etc/cfn/scripts/AmazonCloudWatchAgent.zip",
+                                            " --region ",
+                                            {
+                                                "Ref": "AWS::Region"
+                                            },
+                                            " &&",
+                                            " chown root:root /etc/cfn/scripts/AmazonCloudWatchAgent.zip &&",
+                                            " chmod 700 /etc/cfn/scripts/AmazonCloudWatchAgent.zip"
+                                        ]
+                                    ]
+                                }
+                            },
+                            "02-extract-cloudwatch-agent": {
+                                "command": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "yum -y install unzip &&",
+                                            "unzip /etc/cfn/scripts/AmazonCloudWatchAgent.zip -d /etc/cfn/scripts/aws-cw-agent"
+                                        ]
+                                    ]
+                                }
+                            },
+                            "10-install-cloudwatch-agent": {
+                                "command": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            " bash -xe install.sh &&",
+                                            " systemctl enable amazon-cloudwatch-agent.service &&",
+                                            " systemctl start amazon-cloudwatch-agent.service &&",
+                                            " /opt/aws/amazon-cloudwatch-agent/bin/",
```

Comment:
I think it would be more clear here to have the command on the same line as the path:

```
" /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl",
```